app/helpers/claims_helper.rb

```diff
@@ -35,9 +35,7 @@ def show_message_controls?(claim)
 
   def messaging_permitted?(message)
     return true if message.claim_action.present?
-    if current_user_is_external_user?
-      return Claims::StateMachine::VALID_STATES_FOR_REDETERMINATION.exclude?(message.claim.state)
-    end
+    return !message.claim.redeterminable? if current_user_is_external_user?
```

Comment:
This could be:

```suggestion
    current_user_is_external_user? && !message.claim.redeterminable?
```

and the default `false` on line 39 is unnecessary (but Github will not let me include it in the suggestion). However, you may consider it clearer as it is. No problem either way.